conans/client/tools/oss.py

```diff
@@ -292,6 +284,29 @@ def get_osx_version_name(version):
         elif version.minor() == "10.0.Z":
             return "Cheetha"
 
+    @staticmethod
+    def get_aix_architecture():
+        processor = platform.processor()
+        if "powerpc" in processor:
+            kernel_bitness = OSInfo().get_aix_conf("KERNEL_BITMODE")
+            if kernel_bitness:
+                return "ppc64" if kernel_bitness == "64" else "ppc32"
+        elif "rs6000" in processor:
+            return "ppc32"
+
+        return None
```

Comment:
I think this return is not-needed? e.g. `get_solaris_architecture` doesn't have it

Reply:
removed